app/javascript/view_renewals.js

```diff
@@ -0,0 +1,7 @@
+import renewal from "./submission_handling/renewal";
+
+const renewals = () => {
+    renewal();
+};
+
+export default renewals;
```

Comment:
Are you planning on asyncing the "getting" of checkouts later then? I see that the display of checkouts is still syncronously done.

Reply:
I see. This is going to be in https://github.com/psu-libraries/myaccount/issues/196, I forgot about that separate issue.